src/components/Checklist/Checklist.stories.tsx

```diff
@@ -20,6 +20,9 @@ export default {
             control: { type: 'radio' },
         },
     },
+    args: {
```

Comment:
if you add the args here, there is no need to add it to every story unless you're changing the value.

Reply:
Removed unnecessary args.